pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -111,7 +111,7 @@ func (m *managerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAd
 	return lifecycle.PodAdmitResult{
 		Admit:   false,
 		Reason:  reason,
-		Message: message,
+		Message: getMessage(api.ResourceMemory),
```

Comment:
this could be memory or disk based on node condition.  we need to check for either hasNodeCondition() for NodeMemoryPressure or NodeDiskPressure.